src/app/static/src/app/components/plots/bar/utils.ts

```diff
@@ -109,14 +110,14 @@ export const plotDataToChartData = function (plotData: PlotData,
         }
         dataset.data[labelIdx] = value;
 
-        dataset.errorBars[xAxisLabel] = {};
+        dataset.errorBars![xAxisLabel] = { minus: undefined, plus: undefined };
```

Comment:
this is a necessary evil XD just so the type is happy, i mean we could make them optional keys but i think that would require more changes and assertions because the minus and plus keys are always defined after this